### ISSUE TITLE
0.3.4: failure on collectstatic (jqtree-circle.png)

### ISSUE THREAD

**Issue by generalov** [mbraak/django-mptt-admin#126]
I got a trace after upgrading django-mptt-admin from 0.3.3 to 0.3.4 on `manage.py collectstatic`:
```
Post-processing 'django_mptt_admin/django_mptt_admin.css' failed!

ValueError: The file 'django_mptt_admin/jqtree-circle.png' could not be found with <django.contrib.staticfiles.storage.CachedStaticFilesStorage object
```

It seems the `jqtree-circle.png` is missed.

**Comment by mbraak** [comment]
Thanks for reporting this. I can reproduce this error in the example project.

**Comment by mbraak** [comment]
This is now fixed in the master branch

**Comment by mbraak** [comment]
Fixed in new version 0.3.5
